Lib/fontTools/feaLib/builder_test.py

```diff
@@ -361,6 +363,92 @@ def test_lookup_inside_feature_aalt(self):
             "Lookup blocks cannot be placed inside 'aalt' features",
             self.build, "feature aalt {lookup L {} L;} aalt;")
 
+    def test_extensions(self):
+        class ast_BaseClass(ast.MarkClass):
+            def asFea(self, indent=""):
+                return ""
+
+        class ast_BaseClassDefinition(ast.MarkClassDefinition):
+            def asFea(self, indent=""):
+                return ""
+
+        class ast_MarkBasePosStatement(ast.MarkBasePosStatement):
+            def asFea(self, indent=""):
+                if isinstance(self.base, ast.MarkClassName) :
+                    res = ""
+                    for bcd in self.base.markClass.definitions :
+                        if res != "" : res += "\n{}".format(indent)
```

Comment:
I'd prefer to break `if` statements on different lines -- no matter how short their are--, for the same consistency/pep8 reasons.